### DISCUSSION TITLE
Acerca de la configuración

### DISCUSSION THREAD

**Discussion by jparadaa** [AndresRamos/ARSoftware.Contpaqi.Comercial.Api#5] [Q&A] [answered]
Que tal Andrés,
Para seguir el proceso de instalación/configuración del API es necesario realizarlo en el servidor (Contpaq) o es posible desde terminales?.

Saludos,
Javier

**Comment by AndresRamos** [comment, marked as answer]
No es un requerimiento que se instale y se ejecute el sincronizador en el servidor de CONTPAQi. El sincronizador puede instalarse en otra terminal o servidor, siempre y cuando este se encuentre en la misma red y tenga acceso al servidor de SQL.

Es necesario que la terminal tenga instalado el sistema de CONTPAQi Comercial Premium. Como resumen, si el sistema de CONTPAQi Comercial Premium funciona en la terminal, el sincronizador también debe funcionar.

**Comment by AndresRamos** [comment]
Gracias por tu pregunta, agregue esta respuesta en la documentación de [Requerimientos](https://github.com/AndresRamos/ARSoftware.Contpaqi.Comercial.Api.Community/wiki/Requerimientos) en la sección [Preguntas Frecuentes](https://github.com/AndresRamos/ARSoftware.Contpaqi.Comercial.Api.Community/wiki/Requerimientos#preguntas-frecuentes).
